### DISCUSSION TITLE
Welcome to ti-challenge-bot Discussions!

### DISCUSSION THREAD

**Discussion by 0xPoe** [ti-community-infra/ti-challenge-bot#224] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oceanlvr** [comment]
Mark. 
Hi! I'm @oceanlvr, and I'm a senior student. I hope to participate in contributing to open source, and I am still a novice in front-end and open source.
PingCAP’s partners are very friendly and hope to learn a lot from them.
